init/onboard-msisdn-oracle/insert_MSISDNS

```diff
@@ -0,0 +1,14 @@
+#! /bin/bash
```

Comment:
Do you mind giving this file a .sh extension maybe something like insert_msisdn_into_participantmno.sh? And does it rely on bash specifically? Otherwise /bin/sh.

Reply:
```suggestion
#!/usr/bin/env sh
```
At a glance, doesn't look like it needs `bash`. Good suggestion. Also, _always_ `/usr/bin/env $shell`.